test/test_proxy_tensor.py

```diff
@@ -806,7 +806,8 @@ def f(x):
         shape_env = self._test_dynamic(f, [(3, 4)], test_inputs)
         self.assertTrue(shape_env.evaluate_guards_for_args(torch.randn(4, 5)))
         self.assertFalse(shape_env.evaluate_guards_for_args(torch.randn(25, 5)))
-        assert len(shape_env.guards) == 1
+        # one guard for size/stride contiguity, and one substantive guard
+        assert len(shape_env.guards) == 2, "\n" + shape_env.format_guards()
```

Comment:
self.assertEqual

Reply:
Plain assert is intentional; fir some reason assertEqual is suppressing msg